packages/ui/src/walkthrough-card/WalkthroughCard.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+
+// 3rd party
+import {
+  Link as ReactAriaLink,
+  LinkProps as ReactAriaLinkProps,
+} from "react-aria-components";
+
+import { WalkthroughInfo } from "@repo/data/useWalkthroughData";
+import { URL_WALKTHROUGH_HREF } from "@repo/constants/src/urls";
+import "./WalkthroughCard.css";
+
+export interface WalkthroughCardProps extends ReactAriaLinkProps {
+  id: string;
+  data: WalkthroughInfo;
+  walkthroughId: string;
+}
+
+export default function WalkthroughCard({
+  id,
+  data,
+  walkthroughId,
+}: WalkthroughCardProps) {
+  return (
+    <ReactAriaLink
+      className="ui-WalkthroughCard--CardContainer"
+      href={`${URL_WALKTHROUGH_HREF}/${walkthroughId}`}
+      aria-label={`${id}-walkthrough-card`}
```

Comment:
I like this as it doesn't read the whole card content aloud but I think we may need more context for the user. This is where I always struggle on determining enough context without too much. Something like:

`walkthrough ${id} - ${data.title}` ?

Would read out "walkthrough 9.9.9 [slight pause] Egress from Dwelling Units". Thoughts? Too much text/content?